go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/perses/perses
 
-go 1.22
+go 1.22.0
 
-toolchain go1.22.3
+toolchain go1.22.4
```

Comment:
I'd be curious to have your opinion on that. In some of our repos, we do directly 

``go 1.22.4`` without toolchain. What is really the value of having two lines?